packages/pagination/src/styles/default.module.css

```diff
@@ -0,0 +1,79 @@
+@import '../../../themes/src/default.css';
+
+.pagination {
+  list-style: none;
+
+  & ul {
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-flex-wrap: wrap;
+    -webkit-flex-wrap: wrap;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-align-items: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    gap: var(--gap-m);
+  }
+
+  & button,
+  & a {
+    box-sizing: border-box;
+    background-color: transparent;
+    text-decoration: none;
+    outline: 0;
+    border: 0;
+    margin: 0;
+    padding: 0;
```

Comment:
we could also achieve this  with `all: unset `

Reply:
Amazing tip, thanks!